### PR TITLE
Doc update: Update pipeline info

### PR DESCRIPTION
**Pull request by tmat**

### CODE REVIEW

File: Documentation/Onboarding.md [hunk 1]
```diff
@@ -12,44 +12,36 @@
     2. Add (or copy)
        [Directory.Build.props](https://github.com/dotnet/arcade/blob/main/Directory.Build.props)
        and
-       [Directory.build.targets](https://github.com/dotnet/arcade/blob/main/Directory.Build.targets).
+       [Directory.Build.targets](https://github.com/dotnet/arcade/blob/main/Directory.Build.targets).
     3. Copy `eng\common` from
        [Arcade](https://github.com/dotnet/arcade/tree/main/eng/common)
        into repo.
     4. Add (or copy) the
```
Comment (isadorasophia) [inline Documentation/Onboarding.md:19]: I also needed to add `Publishing.props` or the publishing would not work.

Comment (isadorasophia) [reply Documentation/Onboarding.md:19]: (not sure if here is the right place to note this? but I got lost at the other docs at some point)

File: Documentation/Onboarding.md [hunk 2]
```diff
@@ -12,44 +12,36 @@
...
-       as necessary.
+       files to your `eng\` folder. Adjust the version prefix and prerelease label
+       as necessary. Only include versions for dependencies required by the repository.
     5. Add the following feeds to your nuget.config:
-       * https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json
-       * https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json
-       * https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json
+       * `https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json`
+       * `https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json`
+       * `https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json`
```
Comment (isadorasophia) [inline Documentation/Onboarding.md:28]: I needed to add https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json as well. Is that for preview dotnet only?

File: Documentation/Onboarding.md [hunk 3]
```diff
@@ -12,44 +12,36 @@
...
-  (https://dev.azure.com/dnceng/internal) internal CI and official builds. - See
-  [Onboarding Azure DevOps](AzureDevOps/AzureDevOpsOnboarding.md) and [Moving Official Builds from
-  DevDiv to DncEng](AzureDevOps/MovingFromDevDivToDncEng.md).
+- Set up pipelines
+  - Add pipeline to https://dnceng-public.visualstudio.com/public for public PR validation CI.
+  - Add pipeline to https://dev.azure.com/dnceng/internal for internal validation CI and official builds.
+  - Add pipeline to https://dev.azure.com/dnceng/internal for CodeQL compliance validation.
+
+  See [Onboarding Azure DevOps](AzureDevOps/AzureDevOpsOnboarding.md) and [Moving Official Builds from DevDiv to DncEng](AzureDevOps/MovingFromDevDivToDncEng.md) for details.
 - Onboard onto dependency flow (Darc). - See [Dependency Flow
```
Comment (isadorasophia) [inline Documentation/Onboarding.md:41]: We could note that darc is used for:
1. Push scheduled updates with latest Arcade bits
2. Flow changes between GitHub and Azure DevOps source
3. Subscribe the repo to publish nuget packages to a target feed

I wasn't very sure what darc was for, at first.

File: Documentation/Onboarding.md [hunk 4]
```diff
@@ -12,44 +12,36 @@
     2. Add (or copy)
        [Directory.Build.props](https://github.com/dotnet/arcade/blob/main/Directory.Build.props)
        and
-       [Directory.build.targets](https://github.com/dotnet/arcade/blob/main/Directory.Build.targets).
+       [Directory.Build.targets](https://github.com/dotnet/arcade/blob/main/Directory.Build.targets).
     3. Copy `eng\common` from
```
Comment (isadorasophia) [inline Documentation/Onboarding.md:16]: I had to run chmod +x on all the eng/common .sh scripts. Is that intended? Should I have copied with a different command?

File: Documentation/Onboarding.md (hunk 1 above)
Comment (tmat) [reply Documentation/Onboarding.md:19]: Yeah, the docs are overwhelming. Might use some "refactoring" and cleanup.

Comment (tmat) [reply Documentation/Onboarding.md:19]: @markwilkie

**Review by isadorasophia — APPROVED** [review]

**Review by missymessa — APPROVED** [review]

**Pull request merged**